frontend/src/api/Api.js

```diff
@@ -4,6 +4,10 @@ const baseUri = 'http://localhost:61346/api';
 
 const Api = axios.create({
   baseURL: baseUri,
+  headers: {
```

Comment:
Could you explain this one please?

Reply:
Ask the browser not to cache the results of requests. If the user goes back to the page he was before, the browser pulls cached responses. With this, it doesn't. At least on Firefox, @tmaciulis22 told me that it doesn't work on Chrome